Assets/Scripts/CreatureStats.cs

```diff
@@ -133,19 +140,70 @@ private void DisplayPopup(string text)
         textLabelText = text;
     }
 
+    // Returns true if floats are within 15.0f of each other.
+    // compare Mathf.Approximately.
+    private bool VeryApproximately(float f1, float f2)
```

Comment:
I know Unity named their function Approximately, but that seems like a bad name to me. Could we call it AreApproximate() or something?